bin/lean-gdb.py

```diff
@@ -1,3 +1,11 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+# Released under Apache 2.0 license as described in the file LICENSE.
+#
+# Author: Sebastian Ullrich
```

Comment:
I wasn't even aware this file existed.  This is awesome!  And it even works in CLion!  (I couldn't find any info on how to use it, so for the record, you just have to add `source ..../bin/lean-gdb.py` to your `~/.gdbinit` file.)

How difficult would it be to add pretty-printers for `declaration` and `expr`?